packages/app/src/locale/APP_TARGET.ts

```diff
@@ -1,29 +1,33 @@
 /** This file is meant to trick TypeScript. We're using a webpack plugin to
  * rewrite some imports based on desired locale. This file just exports the
  * type from the NL file so all types still work.
+ *
+ * We still export type TALLLanguages = TNLLocale | TENLocale here because it validates key equality
+ * in TS
+ *
  */
 import locale from './nl.json';
 
-export type TLocale = typeof locale;
+export type Locale = typeof locale;
 
 import en from './en.json';
 import nl from './nl.json';
 
-export type TNLLocale = typeof nl;
-export type TENLocale = typeof en;
+export type NLLocale = typeof nl;
+export type ENLocale = typeof en;
```

Comment:
I'm not 100% sure if we reached consensus on this, but in most other places were are camel-casing acronyms just like normal words. I personally believe it's the only way to stay consistent with types and instances. For example `jsonParser` for instance and `JsonParser` for type/class. Otherwise, what are you are going to call them. jSONParser and JSONParser? or jsonParser and JSONParser? It just easier to think of them as regular words.

```suggestion
export type NlLocale = typeof nl;
export type EnLocale = typeof en;
```

Reply:
I would go for camelCasing too. It's much easier to read.